mgr/Conscript-standard

```diff
@@ -792,6 +792,10 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
     } elsif ( $pkg =~ m/^StJetFinder$/ ){
 	$CXXFLAGS .=  " -I".$env1->{ENV}->{OPTSTAR}."/include";
```

Comment:
Should we change
```suggestion
	$CPPFLAGS .=  " -I".$env1->{ENV}->{OPTSTAR}."/include";
```
to ensure that `FASTJET_DIR` overrides `OPTSTAR`?

Reply:
I have absolutely no problem modifying anything in Cons scripts as long as it works 😺 
But it seems unlikely that Cons provides a functionality to override $CPPFLAGS with $CPPPATH contents... Besides, StJetFinder may include something else from $OPTSTAR/include

Reply:
By override I meant that the `-I` that comes earlier will take precedence. My exact change is incorrect, as you pointed out, it should have been targeting CPPPATH.

```suggestion
	$CPPPATH = $env1->{ENV}->{OPTSTAR}."/include".$main::PATH_SEPARATOR.$CPPPATH;
```

> Besides, StJetFinder may include something else from $OPTSTAR/include

Actually, it's safe to say that it will not. IMO we should move OPTSTAR handling to the else branch of the condition.

Reply:
Fine, let's implement the "if-else" logic